test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -28,6 +30,10 @@ public class MaxRequestBufferSizeTests
             "\r\n"
         };
 
+        private readonly Action<ILoggingBuilder> _configureLoggingDelegate;
+
+        public MaxRequestBufferSizeTests(ITestOutputHelper output) =>_configureLoggingDelegate = builder => builder.AddXunit(output);
```

Comment:
Took me a second to realize this is a constructor. imo the arrow-body syntax obscures this a little too much. Would be better to use brackets.